src/dls_python3_skeleton/__main__.py

```diff
@@ -125,6 +126,11 @@ def replace_text(text: str) -> str:
 
 def new(args):
     path: Path = args.path
+
+    package = path.name
```

Comment:
You need this
```python
package = args.package or path.name
```
But this is duplication.

I suggest putting the following in a function and calling it from both `new` and `existing`:
```python
    package = override_package or repo
    valid = VALID_PKG.match(package)
    assert valid, f"'{package}' is not a valid python package name"
```

Reply:
Yes I worked that out - I got a little confused about which args went to which command. 

Do you not want to include typer as part of skeleton? That argparse looks way old fashioned. As the skeleton provider you get to be opinionated about such things.

Reply:
I specifically chose argparse so it added no external dependencies, and I think that's probably the right approach here...

Reply:
I guess it would be the only non-standard library in the runtime dependencies though.

Reply:
I made it DRYer

Reply:
Skeleton likes DRY...